### ISSUE TITLE
Update Gem to support Rails 5.1

### ISSUE THREAD

**Issue by dg-ratiodata** [weppos/tabs_on_rails#37]
Heya,

would you mind to update your gem to support Rails 5.1?

Currently you implemented its functionality by using `before_filter`. Those are deprecated are going to be removed with rails 5.1. They need to be replaced by `before_action`.

```
DEPRECATION WARNING: before_filter is deprecated and will be removed in Rails 5.1. Use before_action instead. (called from set_tab at /home/user/.rvm/gems/ruby-2.3.0/bundler/gems/tabs_on_rails-01ba07eca6fb/lib/tabs_on_rails/action_controller.rb:106)
```

**Comment by silviorelli** [comment]
https://github.com/weppos/tabs_on_rails/pull/38

**Comment by weppos** [comment]
Closed by ca41c7b2a6486418094c21959ca3aab9d3cb0420
